plugins/obs-webrtc/whip-output.cpp

```diff
@@ -92,23 +93,28 @@ void WHIPOutput::ConfigureAudioTrack(std::string media_stream_id,
 {
 	auto media_stream_track_id = std::string(media_stream_id + "-audio");
 
+	uint32_t ssrc = base_ssrc;
```

Comment:
Is this variable necessary? I guess it helps with readability a bit when you look at the companion video function.

Reply:
It's for code readability mainly, yes. I didn't want there to be confusion about the method being used for SSRC creation. As you can see in the video track function, it's the same style.